api/document/js/__tests__/commands.test.ts

```diff
@@ -41,7 +41,9 @@ describe('appendNearBlock()', () => {
     const selection = new TextSelection(pathToResolvedPos(
       doc,
       // Inside the 'bbb' paragraph
-      [new NthType(1, 'para'), 'inline', 'b'.length],
+      new NthType(1, 'para'),
+      'inline',
+      'b'.length,
```

Comment:
I was throwing around the idea of whether it'd be more readable to have an `NthChar` class analogous to your `NthType` class.  I think it might be a bit more readable, but it'd also be more verbose... 🤷‍♂️